pavelib/assets.py

```diff
@@ -35,6 +35,15 @@
 THEME_SASS_DIRECTORIES = []
 SASS_LOAD_PATHS = ['common/static', 'common/static/sass']
 
+# A list of NPM installed libraries that should be copied into the common
+# static directory.
+NPM_INSTALLED_LIBRARIES = [
+    'underscore/underscore.js'
+]
+
+# Directory to install static vendor files
+STATIC_VENDOR_DIRECTORY = 'common/static/common/js/vendor'
```

Comment:
That's a great point. At the moment openedx is a purely Python package, so has no static assets. It would be good to move all this over, but maybe not now.

FYI, these vendor files get bundled in production so nobody will see the ``common`` paths to them. So it shouldn't be a big deal to move them in the future.